### PR TITLE
Ukpai/double signer

### PR DESCRIPTION
**Pull request by hannydevelop**

### CODE REVIEW

File: cellar_rebalancer/src/collector/poller.rs
Comment (EricBolten) [inline cellar_rebalancer/src/collector/poller.rs:126]: I think this is the last important thing we should be sure to update before merging. Not sure there's anything in the spec as to how large this should be but generating a random 8 character string using the OS randomness should be sufficient I'd imagine.

File: cellar_rebalancer/src/collector/poller.rs
Comment (EricBolten) [inline cellar_rebalancer/src/collector/poller.rs:264]: Ideally we delay for at least a block here before breaking and proceeding to commit, but this isn't a blocker for merge.

**Review by cbrit — APPROVED** [review]

File: cellar_rebalancer/src/collector/poller.rs
Comment (EricBolten) [reply cellar_rebalancer/src/collector/poller.rs:126]: Probably something along the lines of

```
use rand::{distributions::Alphanumeric, rngs::OsRng};

...

salt: OsRng
    .sample_iter(&Alphanumeric)
    .take(8)
    .map(char::from)
    .collect();
```

**Review by EricBolten — APPROVED** [review]
Going to merge to unblock parallel development and will follow up to fix any remaining issues in our next set of changes.

**Pull request merged**
